application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +53,22 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+                logger.info("This is a Error handler of DM");
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
+                logger.debug(err.toString());
+                logger.error(message.toString());
+                if (event.getAuthor().isBot()) {
+                    logger.info("Ready to handle Error is bot");
+                    return;
+                }
+                warnUser(message).queue((res) -> {
+                    logger.info(res.toString());
+                    res.delete().queueAfter(1, TimeUnit.MINUTES);
+                }, (error) -> {
+                    logger.error(error.toString());
+                });
```

Comment:
what is this for? remove it

Reply:
Just Added it In case of any new Errors. Isn't it a good manner to handle Error preventing crash?

Reply:
it is, but not for 1 in a million cases. Also bot won't crash from one error

Reply:
Ok removed the next push will have it